girder/models/file.py

```diff
@@ -37,6 +37,10 @@ def initialize(self):
         self.resourceColl = 'item'
         self.resourceParent = 'itemId'
 
+        self.exposeFields(level=AccessType.READ, fields=(
+            "_id", "mimeType", "itemId", "exts", "name", "created",
+            "assetstoreId", "creatorId", "size"))
```

Comment:
@opadron let's expose `assetstoreId` only under `AccessType.SITE_ADMIN`; assetstores are a detail of the system that is really only exposed to administrators.